generators/app/templates/_package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "build": "npm run lint && npm run test && npm run compile",
     "lint": "eslint src",
-    "test": "nyc mocha --compilers js:babel-register --recursive",
+    "pretest": "npm run lint",
```

Comment:
Hey @rkmax because of this maybe we can change the ``build`` command to just be ``npm run test && npm run compile``?